modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -376,6 +376,343 @@ Additionally, the user requires some `ReadOnly` permissions, and some of the rol
+`Host.Config.Storage`
+`Resource.AssignVMToPool`
+`VApp.AssignResourcePool`
+`VApp.Import`
+`Read Only`
+`VirtualMachine.Config.AddNewDisk`
+
+|vSphere Datastore
+|Always
+|
+[%hardbreaks]
+`Datastore.AllocateSpace`
+`Datastore.Browse`
+`Datastore.FileManagement`
+`Read Only`
+`InventoryService.Tagging.ObjectAttachable`
+
+|vSphere Port Group
+|Always
+|
+[%hardbreaks]
+`Network.Assign`
+`Read Only`
+
+|Virtual Machine Folder
+|Always
+|
+[%hardbreaks]
+`InventoryService.Tagging.ObjectAttachable`
+`Read Only`
+`Resource.AssignVMToPool`
+`VApp.Import`
+`VirtualMachine.Config.AddExistingDisk`
+`VirtualMachine.Config.AddNewDisk`
+`VirtualMachine.Config.AddRemoveDevice`
+`VirtualMachine.Config.AdvancedConfig`
+`VirtualMachine.Config.Annotation`
+`VirtualMachine.Config.CPUCount`
+`VirtualMachine.Config.DiskExtend`
+`VirtualMachine.Config.DiskLease`
+`VirtualMachine.Config.EditDevice`
+`VirtualMachine.Config.Memory`
+`VirtualMachine.Config.RemoveDisk`
+`VirtualMachine.Config.Rename`
+`VirtualMachine.Config.ResetGuestInfo`
+`VirtualMachine.Config.Resource`
+`VirtualMachine.Config.Settings`
+`VirtualMachine.Config.UpgradeVirtualHardware`
+`VirtualMachine.Interact.GuestControl`
+`VirtualMachine.Interact.PowerOff`
+`VirtualMachine.Interact.PowerOn`
+`VirtualMachine.Interact.Reset`
+`VirtualMachine.Inventory.Create`
+`VirtualMachine.Inventory.CreateFromExisting`
+`VirtualMachine.Inventory.Delete`
+`VirtualMachine.Provisioning.Clone`
+`VirtualMachine.Provisioning.MarkAsTemplate`
+`VirtualMachine.Provisioning.DeployTemplate`
+
+|vSphere vCenter Datacenter
+|If the installation program creates the virtual machine folder. For UPI, VirtualMachine.Inventory.Create and VirtualMachine.Inventory.Delete privileges are optional if your cluster does not use the Machine API.
+|
+[%hardbreaks]
+`InventoryService.Tagging.ObjectAttachable`
+`Resource.AssignVMToPool`
+`Read Only`
+`VApp.Import`
+`VirtualMachine.Config.AddExistingDisk`
+`VirtualMachine.Config.AddNewDisk`
+`VirtualMachine.Config.AddRemoveDevice`
+`VirtualMachine.Config.AdvancedConfig`
+`VirtualMachine.Config.Annotation`
+`VirtualMachine.Config.CPUCount`
+`VirtualMachine.Config.DiskExtend`
+`VirtualMachine.Config.DiskLease`
+`VirtualMachine.Config.EditDevice`
+`VirtualMachine.Config.Memory`
+`VirtualMachine.Config.RemoveDisk`
+`VirtualMachine.Config.Rename`
+`VirtualMachine.Config.ResetGuestInfo`
+`VirtualMachine.Config.Resource`
+`VirtualMachine.Config.Settings`
+`VirtualMachine.Config.UpgradeVirtualHardware`
+`VirtualMachine.Interact.GuestControl`
+`VirtualMachine.Interact.PowerOff`
+`VirtualMachine.Interact.PowerOn`
+`VirtualMachine.Interact.Reset`
+`VirtualMachine.Inventory.Create`
+`VirtualMachine.Inventory.CreateFromExisting`
+`VirtualMachine.Inventory.Delete`
+`VirtualMachine.Provisioning.Clone`
+`VirtualMachine.Provisioning.DeployTemplate`
+`VirtualMachine.Provisioning.MarkAsTemplate`
+`Folder.Create`
+`Folder.Delete`
+|===
+====
+
+.Minimum permissions for post-installation management of components
```

Comment:
Post installation here applies to IPI and UPI?

Reply:
yes